runtime/gc_glue_java/CollectorLanguageInterfaceImpl.cpp

```diff
@@ -677,45 +677,50 @@ MM_CollectorLanguageInterfaceImpl::scavenger_switchConcurrentForThread(MM_Enviro
 
 	if (_extensions->scavenger->isConcurrentInProgress()) {
 		Assert_MM_true(NULL != _extensions->scavenger);
```

Comment:
while changing the file, please use _extensions->isConcurrentScavengerInProgress() and remove the assert